docs/experimental-features.md

```diff
@@ -181,7 +181,10 @@ Modify your ipfs config:
 ipfs config --json Experimental.FilestoreEnabled true
 ```
 
-And then pass the `--nocopy` flag when running `ipfs add`
+Restart ipfs node. Otherwise `ipfs add` will not complain and `ipfs filestore`
+will raise error.
+
+Then pass the `--nocopy` flag when running `ipfs add`
 
```

Comment:
As written, this is a bit awkward. How about something like:

> Then restart your IPFS node to reload your config.
>
> Finally, when adding files with `ipfs add`, pass the `--nocopy` flag to use the filestore instead of *copying* the files into your local IPFS repo.

Reply:
Hi, I push the above modification to the new pr: https://github.com/ipfs/go-ipfs/pull/5927. 
I have to create a new pr because the permission of ipbit is changed. Sorry for inconvenience.